src/job-feed/JobList.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { GET_JOBS } from '../sources';
+import Job from './Job';
+import './styles.css';
+
+const JobList = () => {
+  const [jobList, setJobList] = useState([]);
+  const {
+    data, error,
+  } = useQuery(GET_JOBS);
+
+  useEffect(() => {
+    if (data) {
+      setJobList(data.jobs);
+    }
+  }, [data]);
+
+  return (
+    <div className="jobs-container">
+      {error ? null : (jobList && jobList.map((job) => (
+        <Job
+          key={job.id}
+          companyName={job.company.name}
```

Comment:
When reading values deep in the chain is a good practice to check first if they exists. For example if the company name does not exists, your whole component will crash. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
fixed and improved for jobList && jobList.map to jobList?.map